style.css

```
@@ -31,6 +31,54 @@
     display: none;
   }
 
+  #burgerMenu {
+    display: none;
+    padding: 0;
+    width: 100vw;
+    height: 755px;
+    background-color: #3c3a39;
+  }
+
+  .burgerMenuStyle {
+    display: flex;
+    list-style: none;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    height: 240px;
+  }
+
+  #exitButton {
+    position: relative;
+    left: 150px;
+    background-color: transparent;
+    background-image: url(images/exit.png);
+    width: 24px;
+    height: 24px;
+    border: none;
+  }
+
+  .porfolio-desk {
+    position: relative;
+    right: 100px;
+    color: #fff5e1;
+    text-decoration: none;
+  }
+
+  .about-desk {
+    position: relative;
+    right: 110px;
+    color: #fff5e1;
+    text-decoration: none;
+  }
+
+  .contact-desk {
+    position: relative;
+    right: 100px;
+    color: #fff5e1;
+    text-decoration: none;
+  }
```

```suggestion
  .burgerMenuStyle li  {
    width: 80%;
    border-bottom: 2px solid #6f6c6b;
    padding: 15px;
  }

  .burgerMenuStyle li > a {
    color: #fff5e1;
    text-decoration: none;
  }
```

- [ ] Kindly notice the color of the options bar in the Figma design, Please use the exact color in your design.
- [ ] Instead of styling the options one by one, it would be nice to style them together to follow HTML & CSS best practices